app/assets/javascripts/application.js

```diff
@@ -16,3 +16,13 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+def reverse_string(string)
+  new_string = []
```

Comment:
Identifier 'new_string' is not in camel case.<br>Missing semicolon.<br>'new_string' is not defined.